src/components/VulnerabilityEdgePrompt.js

```diff
@@ -1,39 +1,174 @@
 import React from 'react';
```

Comment:
If this is going to become a bigger component we could put it into it's own "VulnerabilityEdgePrompt" folder under components.

Reply:
I believe that editing this component is not expected. This is consumed by other components as it is.

Reply:
THis component is used only if its a edge host.
As soon we start to port our code, this file should not exists.